<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RollResources</class>
 <widget class="QWidget" name="RollResources">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>160</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>274</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="message">
     <property name="text">
      <string>You rolled xx which will give you x [recourses]</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Tool1Label">
         <property name="text">
          <string>+x</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Tool2Label">
         <property name="text">
          <string>+x</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Tool3Label">
         <property name="text">
          <string>+x</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="Tool1Button">
         <property name="text">
          <string>Add tool 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Tool2Button">
         <property name="text">
          <string>Add tool 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Tool3Button">
         <property name="text">
          <string>Add tool 3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="tempTool1Label">
         <property name="text">
          <string>+x</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="tempTool2Label">
         <property name="text">
          <string>+x</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="tempTool3Label">
         <property name="text">
          <string>+x</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="tempTool3Button">
         <property name="text">
          <string>Add temp tool</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tempTool2Button">
         <property name="text">
          <string>Add temp tool</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tempTool1Button">
         <property name="text">
          <string>Add temp tool</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Okay</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
